docs/src/content/en/docs/workflows/overview.mdx

```diff
@@ -1,18 +1,30 @@
 ---
 title: "Handling Complex LLM Operations | Workflows | Mastra"
-description: "Workflows in Mastra help you orchestrate complex sequences of operations with features like branching, parallel execution, resource suspension, and more."
+description: "Workflows in Mastra help you orchestrate complex sequences of tasks with features like branching, parallel execution, resource suspension, and more."
 ---
 
 import { Steps } from "nextra/components";
 
 # Workflows overview
 
-Workflows let you define and orchestrate complex sequences of tasks as **typed steps** connected by data flows. Each step has clearly defined inputs and outputs validated by Zod schemas.
-
-A workflow manages execution order, dependencies, branching, parallelism, and error handling — enabling you to build robust, reusable processes. Steps can be nested or cloned to compose larger workflows.
+Workflows let you define and orchestrate complex sequences of tasks as **typed steps** connected by data flows. While a single agent may be suitable for simple tasks, real-world applications often involve multiple steps with different logic, tools, external services, or context requirements.
 
 ![Workflows overview](/image/workflows/workflows-overview.jpg)
 
+## Real-world example
+
+For example, you might want to:
+
+- Handle a user question like “Can I return my last order?”
```

Comment:
When I read this, I thought I was reading a list of example scenarios. I think I read the h2 as "real-world exampleS" and I associated ul with lists, but these read more like steps. These mistakes are easy to make when scanning. Maybe use a ol or, better yet, a diagram